app/graphql/types/base_object.rb

```diff
@@ -34,6 +34,15 @@ def authorized?(object, context)
       def enforce_rbac(*permissions)
         @rbac_permissions = permissions
       end
+
+      # This the field to be stored in the GraphQL fragment cache with a `title/` prefix
+      # Please note that only static and context-free fields, such as SSG content can be
+      # stored in this cache and that it gets cleared upon a successful SSG import.
+      def cached_static_field(title, type, **args)
+        field(title, type, **args.merge(
+          cache_fragment: { path_cache_key: title.to_s, cache_key: :object }
```

Comment:
Wouldn't the `cache_key` had a literal of `object`?

Reply:
Nope, this is not in that context.

Reply:
Okay, so this would use the object's `cache_key` implementation, in our case from ActiveRecord's models.